src/components/AudioTrack/layouts/Row.vue

```diff
@@ -82,6 +84,11 @@ export default {
       isSmall,
```

Comment:
```suggestion
      isSmall,
      
      separatorCharacter: '•',
```

Reply:
Great suggestion. Even if it's consistent across languages let's put that decision in the hands of our translators.